index.html

```diff
@@ -65,6 +65,217 @@ <h3><a> Let's connect </a> </h3>
         </div>
     </section>
 
+    <section class="portfolio-1">
```

Comment:
- [ ] Kindly add `display: grid;` to the main container of that has the projects 

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![bgc](https://user-images.githubusercontent.com/69638013/153730857-0c9f472c-ec23-4c7b-99fe-504e3df2b4f8.PNG)  | ![bgcd](https://user-images.githubusercontent.com/69638013/153730847-437997d3-3351-4ae4-a4c2-9876ced8acc9.PNG)



- [ ] Kindly add the right background color to the section and each project cards, this will make your work look more like the original design 
- [ ] kindly add border and border outline to each project card 
- [ ] Kindly reduce spacing li's in skills project div 💪🏿

Reply:
- [ ] Kindly implement all changes requested from previous review, please tag @prolajumokeoni if you have any questions.